ui/test-view/ui/Test.jsx

```diff
@@ -48,25 +46,27 @@ class BugCountComponent extends React.Component {
 
   render() {
     return (
-      <td className="bug-count" onClick={this.onClick}>
-        {// TODO: Clean this up
-        // eslint-disable-next-line no-nested-ternary
-        this.props.test.bugs === undefined ? (
-          <FontAwesomeIcon
-            icon={faMinus}
-            title="Click to expand and fetch bugs"
-          />
-        ) : Object.keys(this.props.test.bugs).length > 0 ? (
-          Object.keys(this.props.test.bugs).length
-        ) : (
-          <Badge
-            size="sm"
-            color="danger"
-            style={{ fontWeight: 400, fontSize: '.8rem', margin: '0 .5rem' }}
-          >
-            0
-          </Badge>
-        )}
+      <td className="bug-count">
```

Comment:
This is totally fine like this.  But, as I said, I'll be removing this code at some point soon anyway.  :)